copier/tools.py

```diff
@@ -103,15 +103,29 @@ def to_nice_yaml(data: Any, **kwargs) -> str:
     return result or ""
 
 
+def get_env(
+    envops: EnvOps,
+    filters: Optional[Dict[str, Callable]] = None,
+    paths: Optional[Union[str, Iterable[str]]] = None,
+    **kwargs: Any,
+):
+    loader = FileSystemLoader(paths) if paths else None
+    # We want to minimize the risk of hidden malware in the templates
+    # so we use the SandboxedEnvironment instead of the regular one.
+    # Of couse we still have the post-copy tasks to worry about, but at least
+    # they are more visible to the final user.
+    env = SandboxedEnvironment(loader=loader, **envops.dict(), **kwargs)
+    env.filters.update(filters or {})
```

Comment:
As said in another comment, maybe we should add filters here by default.

Reply:
After reading more code, yes, this seems the best place for that.